CONTRIBUTING.md

```diff
@@ -56,7 +56,7 @@ All schemas should have unit tests, which primarily should be validation tests.
 
 ### Resources
 
-Flask-RESTful uses the term "resources" in place of "controllers" in an MVC framework and that is also the term we use in this project.
+Flask-RESTful used the term "resources" in place of "controllers" in an MVC framework and that is also the term we use in this project. If your familiar with Django this would be a View.
```

Comment:
```suggestion
Flask-RESTful used the term "resources" in place of "controllers" in an MVC framework and that is also the term we use in this project. If you're familiar with Django this would be a View.
```